docs/source/data-model.rst

```diff
@@ -62,6 +82,10 @@ For each type, we will show:
 Run Start Document
 ------------------
 
+Again, a 'run start' document marks the beginning of the run. It comprises
```

Comment:
Here we go again: `'run start'`.  There is no such `run start` document in my stream.  How is that different from a `'start'` document?  Ah, yes, they are the same.

Reply:
run 'start'

Reply:
new PR?

Reply:
op cit